### DISCUSSION TITLE
ccf-validation-tools/Brain/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#298] [General]
# ccf-validation-tools/Brain/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Brain/comments/

**Comment by emquardokus** [comment]
Brain_v1.5_DRAFT
https://docs.google.com/spreadsheets/d/1DrsZo9QUgnye6qX9hGKQoXVruRbM3LZPT2tzUy-SixI/edit#gid=2056967441

**Different labels** Fixed for publication, contacting Jeremy Miller to fix in his database for permanent fix.

If the name/label in the ontology contains obsolete, please look into OLS, clicking on the term ID, for its replacement.

FIXED: The term UBERON:0002705 has a different name/label in the source ontology in the following 1 row 102. The name/label in the ASCT+B table is midline nuclear complex and the one in the ontology is midline nuclear group. For reference, the given name/label by SMEs is midline nuclear complex. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED: The term PCL:0015079 has a different name/label in the source ontology in the following 1 row 736. The name/label in the ASCT+B table is Oligo L2-6 OPALIN MAP6D1 primary motor cortex oligodendrocyte (Hsap) and the one in the ontology is Oligo L2-6 OPALIN MAP6D1 primary motor cortex oligodendrocyte precursor cell (Hsap). For reference, the given name/label by SMEs is Oligo L2-6 OPALIN MAP6D1. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED: The term PCL:0015120 has a different name/label in the source ontology in the following 1 row 738. The name/label in the ASCT+B table is Oligo L2-6 OPALIN FTH1P3 primary motor cortex oligodendrocyte (Hsap) and the one in the ontology is Oligo L2-6 OPALIN FTH1P3 primary motor cortex oligodendrocyte precursor cell (Hsap). For reference, the given name/label by SMEs is Oligo L2-6 OPALIN FTH1P3. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED The term PCL:0015119 has a different name/label in the source ontology in the following 1 row 737. The name/label in the ASCT+B table is Oligo L3-6 OPALIN-like ENPP6 primary motor cortex oligodendrocyte (Hsap) and the one in the ontology is Oligo L3-6 OPALIN-like ENPP6 primary motor cortex oligodendrocyte precursor cell (Hsap). For reference, the given name/label by SMEs is Oligo L3-6 OPALIN ENPP6. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
As per https://github.com/hubmapconsortium/ccf-validation-tools/issues/293, I removed the mapping present in line 111, AS5. We can propose to authors if tuber cinereum is more appropriate since it is a related synonym of tuberal region (SME label) and it is part of the hypothalamus (AS4). This mapping did not come from the table of possible mappings that I had sent, so it must have been some sort of automated mapping tool.  I wil contact Jeremy Miller for update in next revision.

**Comment by emquardokus** [comment]
Sent this to Jeremy Miller October 3, 2023 for his consideration; I used the exisiting ASCT+B validation report to look up the remaining brain regions to see if there were exact synonyms to terms in his table.  He replied with thanks and he would confer with David Osumi-Sutherland.  He then updated his table Brain v1.5 with any that he approved of.  The current validation report would suggest he used many of the exact synonym suggestions.  Note the one flagged in #293 was NOT a recommended mapping from my table attached here.  Not sure where that came from, but will inquire.
[new_uberon_terms_Brain-possible-mappings.xlsx](https://github.com/hubmapconsortium/ccf-validation-tools/files/13350708/new_uberon_terms_Brain-possible-mappings.xlsx)

**Comment by aleixpuigb** [comment]
**New terms from other projects (BICAN, CAP)**

CL:4030059 [L2/3 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030059)
CL:4030060 [L2 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030060)
CL:4030061 [L3 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030061)
CL:4030062 [L4/5 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030062)
CL:4030063 [L4 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030063)
CL:4030064 [L5 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030064)
CL:4030065 [L6 intratelencephalic projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030065)
CL:4030067 [L5/6 near-projecting glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030067)
CL:4030068 [L6 intratelencephalic projecting Car3 glutamatergic neuron](http://purl.obolibrary.org/obo/CL_4030068)

**Comment by aleixpuigb** [comment]
New terms
The 461 CT terms with no ID have been added in PCL: https://docs.google.com/spreadsheets/d/1S6IvO7BIoQyh2k9MsGAMvFPTtG9zI202doXjPMu3mOE/edit?usp=sharing

**Comment by aleixpuigb** [comment]
New terms in release v2024-07-10 from BICAN:

L2 marginal neuron (CL:4042012)
Lamp5 Lhx6 neuron (CL:4042013)

**Comment by emquardokus** [comment]
**Brain_v1.7_DRAFT**
https://docs.google.com/spreadsheets/d/1BHmFep0g8vKGu3FVNye7O6-32aC15W-Q2wel1wFYeZ4/edit?gid=2056967441#gid=2056967441

**Different labels**
Slacked Aleix Puig to ask where this new label came from that only recently was flagged and it was "change from last CL release: https://github.com/obophenotype/cell-ontology/issues/2585
Then I would recommend to update the term on the table since it should be stable now."
Emailed Jeremy Miller to confirm and notify of the issue and he said go ahead and fix; FIXED all instances
    The term CL:4023064 has a different name/label in the source ontology in the following 60 rows 434, 435, 436, 437, 438, 439, 440, 441, 442, 443, 444, 445, 446, 447, 448, 449, 450, 451, 452, 453, 454, 619, 620, 621, 622, 623, 624, 625, 626, 627, 628, 629, 630, 631, 632, 633, 634, 635, 636, 637, 638, 639, 640, 641, 642, 643, 644, 645, 646, 647, 648, 649, 650, 651, 652, 653, 654, 655, 656, 657. The name/label in the ASCT+B table is caudal ganglionic eminence derived interneuron and the one in the ontology is caudal ganglionic eminence derived cortical interneuron. For reference, the given name/label by SMEs is CGE interneuron. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by aleixpuigb** [comment]
**Update in CL**
In the next release (January 2025 release), the term CNS interneuron will be obsoleted and replaced by interneuron (CL:0000099). This affects lines 422 to 433.
- For lines 422 to 424, the term (in column W) should be replaced by 'chandelier cell' CL:4023083.
- For lines 425 to 433: 1) the term should be replaced by interneuron (CL:0000099). 2) In the next PCL release, the labels in column AA will be updated, removing CNS from the label (e.g., in row 425 will be 'LLC_267 interneuron (Hsap)'. This will not affect the ID, only the label.
